man/scale_viridis.Rd

```diff
@@ -128,16 +129,25 @@ expand the scale by 5\% on each side for continuous variables, and by
 \item{alpha}{The alpha transparency, a number in [0,1], see argument alpha in
 \code{\link[grDevices]{hsv}}.}
 
-\item{begin}{The (corrected) hue in [0,1] at which the viridis colormap begins.}
+\item{begin, end}{The (corrected) hue in \verb{[0,1]} at which the color map
+begins and ends.}
 
-\item{end}{The (corrected) hue in [0,1] at which the viridis colormap ends.}
+\item{direction}{Sets the order of colors in the scale. If 1, the default,
+colors are ordered from darkest to lightest. If -1, the order of colors is
```

Comment:
```suggestion
colors are ordered from darkest to lightest. If -1, the order of colors are
```

Reply:
This is inherited from the ggplot2 documentation.